presto-hive/src/main/java/com/facebook/presto/hive/ParquetHiveRecordCursor.java

```diff
@@ -884,11 +901,6 @@ public ParquetListEntryConverter(Type prestoType, String columnName, GroupType e
                     columnName,
                     elementType.getFieldCount());
 
-            checkArgument(elementType.getFieldName(0).equals("array_element"),
```

Comment:
you mean a check like this `isElementType(elementType.getType(0), elementType.getName()`? I think it should be OK.